notifier.rst

```diff
@@ -82,7 +83,7 @@ Twilio      ``symfony/twilio-notifier``       ``twilio://SID:TOKEN@default?from=
 
 .. versionadded:: 5.3
 
-    The Iqsms, GatewayApi, Octopush and AllMySms integrations were introduced in Symfony 5.3.
+    The Iqsms, GatewayApi, Octopush, AllMySms and FakeSms integrations were introduced in Symfony 5.3.
```

Comment:
Maybe it would be better to have a `since` column in the table, instead of adding a new `versionadded` every new minor?

Though thats something the maintainers should decide

Reply:
I also think it's a good idea, @OskarStark ?

Reply:
@javiereguiluz I tend to agree and like the idea but the problem is the table width, right now it is already to width